TUM Campus App/CollectionViewLayouts/ColumnsFlowLayout.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ColumnsFlowLayout.swift
+//  Campus
+//
+//  Created by Till on 30.06.18.
+//  Copyright © 2018 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+class ColumnsFlowLayout: NSObject, UICollectionViewDelegateFlowLayout {
```

Comment:
The naming isn't accurate, since this is not a FlowLayout but rather a FlowLayoutDelegate

Reply:
Alright, renamed these classes.